services/movie_session.py

```diff
@@ -26,9 +28,9 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
 
 def update_movie_session(
     session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
+    show_time: Optional[str] = None,
+    movie_id: Optional[int] = None,
+    cinema_hall_id: Optional[int] = None,
 ) -> None:
     movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
The `get()` method is used without handling the `DoesNotExist` exception. Consider using `get_object_or_404()` or wrapping this call in a try-except block to handle the exception and avoid runtime errors.